src/main/java/ch/uzh/ifi/hase/soprafs24/service/GameService.java

```diff
@@ -300,6 +313,26 @@ private void runActiveTurn(Game currentGame, Integer cardId) throws InterruptedE
         }
     }
 
+    public void handleInactivePlayer(Integer gameId) {
+        if (isGameInCalc(gameId)) return;
+        User inactivePlayer = UserContextHolder.getCurrentUser();
+        Game currentGame = inMemoryGameRepository.findById(gameId);
+        if (currentGame != null && inactivePlayer.getUserId().equals(currentGame.getActivePlayer())) {
+            setCardsFaceDown(currentGame, currentGame.getHistory().get(currentGame.getHistory().size()-1));
+            initiateNewTurn(currentGame, false);
+
+            inMemoryGameRepository.save(currentGame);
+
+            WSGameChangesDto wsGameChangesDto = WSGameChangesDto.builder()
+                    .cardsStates(new WSCardsStates(currentGame.getCardCollection().getAllCardStates()))
+                    .gameChangesDto(WSGameChanges.builder().activePlayer(currentGame.getActivePlayer()).build())
+                    .build();
+
+            eventPublisher.publishEvent(new GameChangesEvent( this, gameId, wsGameChangesDto));
+
+        }
+    }
+
```

Comment:
is this method for creating an backup path for mismatches btw frontend & backend?